### PR TITLE
chore: readme major update

### PR DESCRIPTION
**Pull request by nabil-el-houssein**
I added ReVanced logo next to the name in order to be used in the README.md. Moreover, I fixed some grammatical and structural errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
-# 💊 ReVanced Manager
+<p align="center">
+  <img width="800" src="Logo.png">
```
Comment (oSumAtrIX) [inline README.md:2]: This doesn't follow any style guide. Please refer to [this](https://cdn.discordapp.com/attachments/952987428786941952/1067552057940267079/revanced-brand-identity.zip).

File: README.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-The official ReVanced Manager based on Flutter.
 
-> **Warning**: This repository currently has no active maintainer. For that reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.
+### <p align="center">  The official ReVanced Manager based on Flutter</p>
+
+> **Warning**: This repository currently has no active maintainer. For this reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.
 
 ## 🔽 Download
-To download the Alpha version of Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.
+ReVanced Manager is still in early development (ALPHA) . You can install the latest version [here](https://github.com/revanced/revanced-manager/releases/latest).
```
Comment (oSumAtrIX) [inline README.md:11]:
```suggestion
ReVanced Manager is still in early development. You can obtain the latest version [here](https://github.com/revanced/revanced-manager/releases/latest).
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 
 ## 📝 Prerequisites
-1. Android 8 or higher
-2. Does not work on some armv7 devices
-3. [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases) required for YouTube and YouTube Music (Only for non-root)
+1. Android 8 or higher.
+2. [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases) is required for YouTube and YouTube Music (only for non-root).
```
Comment (oSumAtrIX) [inline README.md:15]: This is not a prerequisite.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+
+&nbsp; 🟥 Does not work on some ARMv7 devices
```
Comment (oSumAtrIX) [inline README.md:17]: This is inconsistent markdown. Use lists.

File: README.md [hunk 5]
```diff
@@ -19,19 +23,19 @@ To download the Alpha version of Manager, go [here](https://github.com/revanced/
 For suggestions and bug reports, open an issue [here](https://github.com/revanced/revanced-manager/issues/new/choose).
 
 ## 💭 Discussion
-If you wish to discuss the Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in the Discord server, please note that this thread may be temporary and may be removed in the future.
+If you wish to discuss the Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel in the Discord server. *This thread may be removed in the future.*
```
Comment (oSumAtrIX) [inline README.md:26]:
```suggestion
If you wish to discuss about ReVanced Manager, a thread has been made under the [#development](https://discord.com/channels/952946952348270622/1002922226443632761) channel on the ReVanced Discord server. *This thread may be closed in the future.*
```

File: README.md [hunk 6]
```diff
@@ -19,19 +23,19 @@ To download the Alpha version of Manager, go [here](https://github.com/revanced/
...
 
 
 ## 🌐 Translation
 [![Crowdin](https://badges.crowdin.net/revanced/localized.svg)](https://crowdin.com/project/revanced)
 
-If you wish to translate ReVanced Manager, we're accepting translations on [Crowdin](https://translate.revanced.app)
+We are accepting translations on [Crowdin](https://translate.revanced.app)
```
Comment (oSumAtrIX) [inline README.md:32]:
```suggestion
Translations can be made on [Crowdin](https://translate.revanced.app)
```

File: README.md [hunk 7]
```diff
@@ -19,19 +23,19 @@ To download the Alpha version of Manager, go [here](https://github.com/revanced/
...
 
-## 🛠️ Building Manager from source
-1. Setup flutter environment for your [platform](https://docs.flutter.dev/get-started/install)
-2. Clone the repository locally
-3. Add your github token in gradle.properties like [this](https://github.com/revanced/revanced-manager/blob/docs/docs/5_building-from-source.md)
-4. Open the project in terminal
-5. Run `flutter pub get` in terminal
-6. Then `flutter packages pub run build_runner build --delete-conflicting-outputs` (Must be done on each git pull)
-7. To build release apk run `flutter build apk`
+## 🛠️ Building the Manager from the source code
```
Comment (oSumAtrIX) [inline README.md:34]:
```suggestion
## 🛠️ Building ReVanced Manager from source
```

File: README.md [hunk 8]
```diff
@@ -19,19 +23,19 @@ To download the Alpha version of Manager, go [here](https://github.com/revanced/
...
+1. Setup flutter environment for your [platform.](https://docs.flutter.dev/get-started/install)
+2. Clone the repository locally.
+3. Add your github token in gradle.properties [like this](https://github.com/revanced/revanced-manager/blob/docs/docs/5_building-from-source.md).
+4. Open the project in the terminal.
```
Comment (oSumAtrIX) [inline README.md:38]:
```suggestion
4. Open the project in a terminal.
```

File: README.md [hunk 9]
```diff
@@ -19,19 +23,19 @@ To download the Alpha version of Manager, go [here](https://github.com/revanced/
...
-## 🛠️ Building Manager from source
-1. Setup flutter environment for your [platform](https://docs.flutter.dev/get-started/install)
-2. Clone the repository locally
-3. Add your github token in gradle.properties like [this](https://github.com/revanced/revanced-manager/blob/docs/docs/5_building-from-source.md)
-4. Open the project in terminal
-5. Run `flutter pub get` in terminal
-6. Then `flutter packages pub run build_runner build --delete-conflicting-outputs` (Must be done on each git pull)
-7. To build release apk run `flutter build apk`
+## 🛠️ Building the Manager from the source code
+1. Setup flutter environment for your [platform.](https://docs.flutter.dev/get-started/install)
```
Comment (oSumAtrIX) [inline README.md:35]: Usually, you don't end lists with a punctuation mark.

**Review by oSumAtrIX — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kazimmt) [reply README.md:2]: What is this 😵‍💫😵‍💫

Btw OsuMatrix, Circular logo where?

Comment (oSumAtrIX) [reply README.md]: Masked off the full logo in the zip.

**Comment by kazimmt** [comment]
Leave that logo part for now or follow the brand guide from the zip archive.

**Comment by oSumAtrIX** [comment]
The guide should be followed.

File: README.md (hunk 1 above)
Comment (kazimmt) [reply README.md]: This is not even the logo i updated 🙁
It still the old one

Comment (oSumAtrIX) [reply README.md]: I can't find the updated one, also it's not open source, might want to create a repository for it.

Comment (kazimmt) [reply README.md]: [Here](https://cdn.discordapp.com/attachments/953965039105232906/1070354383159709798/revanced-brand-identity.zip)

File: README.md (hunk 4 above)
Comment (nabil-el-houssein) [reply README.md:17]: Not working on ARMv7 devices is not a prerequisite so I thought that including it in the list doesn't make any sense

File: README.md (hunk 3 above)
Comment (nabil-el-houssein) [reply README.md:15]: I didn't change any prerequisite. These are the ones written in the `flutter` branch.

File: README.md (hunk 1 above)
Comment (nabil-el-houssein) [reply README.md]: Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos

Comment (kazimmt) [reply README.md]: > Sorry for the inconvenience. I chose the logo from the `res` folder in the main code. I will make sure to abide by the official logos.

Logo & typo both

**Comment by nabil-el-houssein** [comment]
Thanks for the zip file as I wasn't able to find the wordmark before. Anyway, I can't seem to find a logo combined with the wordmark for the README. Is there one? Plus, which logo should be used in the README in order to be readable on github in both light and dark mode.

File: README.md (hunk 4 above)
Comment (oSumAtrIX) [reply README.md:17]: In that case it doesn't belong in the section at all and should be moved outside

File: README.md (hunk 3 above)
Comment (oSumAtrIX) [reply README.md:15]: It's not correct in the `flutter` branch.

**Comment by oSumAtrIX** [comment]
The Zip doesn't include a logo with a wordmark. Both have to be used separately and support dark and light theme.

File: README.md [hunk 10]
```diff
@@ -1,16 +1,20 @@
-# 💊 ReVanced Manager
+<p align="center">
+  <img width="800" src="Logo (with text).png">
+</p>
```
Comment (EvadeMaster) [inline README.md:3]: The logo can be improved by changing the font to [Manrope](https://github.com/sharanda/manrope "Manrope font on GitHub"), Open Sans, or similar. See my example design for reference. *you can use this design, there's nothing stopping you :) but [see this.](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)*

ReVanced Manager Valentine - Light mode


ReVanced Manager Valentine - Dark mode

Comment (kazimmt) [reply README.md:3]: Why font needs to change? 
Why not follow the brand guide?

File: README.md (hunk 1 above)
Comment (kazimmt) [reply README.md]: @oSumAtrIX what about this wordmark?
revanced-manager-wordmark

File: README.md (hunk 10 above)
Comment (kazimmt) [reply README.md:3]: revanced-manager-wordmark

Comment (oSumAtrIX) [reply README.md:3]: We currently lack an extensive kit of our current language design. A repository should be created to discuss this matter openly.

Comment (kazimmt) [reply README.md:3]: > Why font needs to change? Why not follow the brand guide?

I just realize Manrope is the font used on ReVanced wordmark. My bad

**Pull request closed without merging**